### DISCUSSION TITLE
Using Toggle Group in a Form

### DISCUSSION THREAD

**Discussion by tskuo** [huntabyte/shadcn-svelte#1131] [Help]
Hi, I am trying to include a toggle group (type = single) inside the form components offered by shadcn-svelte. With the following code, I can see the selected toggle gets updated within the SuperDebug component.
```
<Form.Fieldset {form} name="answer">
<Form.Legend>Toggle A, B, or C</Form.Legend>
  <ToggleGroup.Root bind:value={$formData.answer} class="flex">
    <Form.Control let:attrs>
      <ToggleGroup.Item value="A" {...attrs}><Form.Label>A</Form.Label></ToggleGroup.Item>
    </Form.Control>
    <Form.Control let:attrs>
      <ToggleGroup.Item value="B" {...attrs}><Form.Label>B</Form.Label></ToggleGroup.Item>
    </Form.Control>
    <Form.Control let:attrs>
      <ToggleGroup.Item value="C" {...attrs}><Form.Label>C</Form.Label></ToggleGroup.Item>
    </Form.Control>
  </ToggleGroup.Root>
  <Form.FieldErrors />
</Form.Fieldset>
```
However, when I console log the received form data on the `page.server.ts` side, the `form.data.answer` is an empty string. I am able to get the other components, such as input, radio group, and textarea, working inside the form. I would really appreciate any pointers or examples to get the toggle group working as well. Thank you!

**Comment by michal-weglarz** [comment]
I know it's an old question, but in such cases you can use a hidden input, making sure it has a proper `name`, as hinted [here in Superforms docs](https://superforms.rocks/faq#i-see-the-data-in-form-but-its-not-posted-to-the-server). 

For instance: 
```svelte
<input type="hidden" name="answer" value={$formData.answer} />
```
